custom_components/powercalc/data/signify/915005996601/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
```

Comment:
You may also add the more detailed description (that you used a 40W dummy load etc) here.